src/main/java/com/example/hop_oasis/controller/CartController.java

```diff
@@ -1,93 +1,51 @@
 package com.example.hop_oasis.controller;
 
+import com.example.hop_oasis.dto.CartDto;
 import com.example.hop_oasis.dto.CartItemDto;
 import com.example.hop_oasis.dto.ItemRequestDto;
 import com.example.hop_oasis.model.ItemType;
-import com.example.hop_oasis.dto.CartDto;
 import com.example.hop_oasis.service.CartService;
-import jakarta.servlet.http.Cookie;
-import jakarta.servlet.http.HttpServletRequest;
-import jakarta.servlet.http.HttpServletResponse;
-import jakarta.servlet.http.HttpSession;
 import lombok.RequiredArgsConstructor;
+import lombok.extern.log4j.Log4j2;
 import org.springframework.http.ResponseEntity;
 import org.springframework.web.bind.annotation.*;
 
-import java.util.List;
 
 @RestController
-@RequiredArgsConstructor
 @RequestMapping("/cart")
-@SessionAttributes("cart")
+@RequiredArgsConstructor
+@Log4j2
```

Comment:
You can remove @Log4j2 if you don't log anything

Reply:
done